great_expectations/datasource/fluent/interfaces.py

```diff
@@ -110,7 +110,7 @@ class PartitionerSortingProtocol(Protocol):
     sort_ascending: bool
 
     @property
-    def param_names(self) -> list[str]:
+    def param_names(self) -> Union[list[str], tuple[str]]:
```

Comment:
This type is incorrect; `tuple[str]` indicates a tuple with exactly one string. I think you want `tuple[str, ...]`. (that doesn't solve your type ignore below, though, unfortunately)

Reply:
thanks, that's correct

Reply:
updated in [update return type](https://github.com/great-expectations/great_expectations/pull/10543/commits/b2ea48d4a8b8c575270d0d2ac736609d014ccf01)

Reply:
this actually did solve the type ignore, thanks! 🥳